modules/update-mirror-repository.adoc

```diff
@@ -135,6 +140,13 @@ $ oc apply -f ${REMOVABLE_MEDIA_PATH}/mirror/config/<image_signature_file> <1>
 +
 <1> For `<image_signature_file>`, specify the path and name of the file, for example, `signature-sha256-81154f5c03294534.yaml`.
 
+... Mirror the release image to a separate repository:
++
+[source,terminal]
+----
+$ oc image mirror -a ${LOCAL_SECRET_JSON} ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE} ${LOCAL_REGISTRY}/${LOCAL_RELEASE_IMAGES_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE}
```

Comment:
Looks like we also want [this block][1] initializing `LOCAL_RELEASE_IMAGES_REPOSITORY`

[1]: https://github.com/openshift/openshift-docs/blame/a9eea486ac20571635f3eda6a07eb3068c1de854/modules/update-service-mirror-release.adoc#L66-L74

Reply:
Done, thanks. Let me know if I'm missing anything else!